watertap/core/plugins/solvers.py

```diff
@@ -158,7 +158,9 @@ def _scale_constraints(self, blk):
                     self._cleanup()
                     raise RuntimeError(
                         "Error in AMPL evaluation.\n"
-                        "Run ipopt with halt_on_ampl_error=yes and symbolic_solver_labels=True to see the affected function."
+                        "Re-run ipopt with:\n"
+                        '1. solver options = {"halt_on_ampl_error" : "yes", "nlp_scaling_method" : "gradient-based"\n'
```

Comment:
I am forgetting- should `nlp_scaling_method` be what you have here or `user-scaling`? What happens if you don't specify gradient -based scaling?

Reply:
This option ensures watertap's own scaling gets skipped and the model gets handed directly to Ipopt. This is what we want to see the problematic constraint / expression.

Reply:
I just thought I remembered this working regardless of that option so maybe I'm not recalling correctly.

Reply:
It certainly does in IDAES or if you just use `SolverFactory("ipopt")`.